lib/cartopy/mpl/geoaxes.py

```diff
@@ -1978,11 +2006,11 @@ def add_wms(self, wms, layers, wms_kwargs=None, **kwargs):
 
 def _trigger_patch_reclip(event):
     """
-    Define an event callback for a GeoAxes which forces the outline and
-    background patches to be re-clipped next time they are drawn.
+    Define an event callback for a GeoAxes which forces the background patch to
+    be re-clipped next time it is drawn.
 
     """
     axes = event.axes
     # trigger the outline and background patches to be re-clipped
-    axes.outline_patch.reclip = True
+    axes.spines['geo'].stale = True
```

Comment:
The docstring only talks about forcing the background patch to be reclipped. Do we really need to set stale here?

Reply:
Originally, no matter which had `reclip=True`, [both would be updated](https://github.com/SciTools/cartopy/pull/1213/files/fbc8de276a37679e7b14657a0322b0faab76cdb9#diff-b599a1ae2427ad1c63351afff7e4e9c5R411). Setting `stale` now is equivalent to setting what `reclip` did except each thing happens independently.

It doesn't seem like this is necessary now (at least from tests) and I don't remember exactly why I left it, but I believe this will be necessary when `backgroud_patch` is removed. I will try again when I get to working on that once more.